packages/editor/src/components/page-attributes/parent.js

```diff
@@ -65,17 +65,17 @@ const applyWithDispatch = withDispatch( ( dispatch ) => {
 const applyWithAPIDataItems = withAPIData( ( { postType, postId } ) => {
 	const isHierarchical = get( postType, [ 'hierarchical' ], false );
 	const restBase = get( postType, [ 'rest_base' ], false );
-	const queryString = stringify( {
+	const queryString = {
```

Comment:
`queryString` isn't the most accurate name for this variable assignment any longer. Maybe just `query`.